templates/base.html.twig

```diff
@@ -1,127 +1,186 @@
-                                <li class="menu-item"><a href="https://github.com/sfu-dhil/circus">GitHub</a></li>
-                                <li class="menu-item last">{{ knp_menu_render('nines_user_nav') }}</li>
-                                <li class="menu-item last">
-                                    <a href="{{ path('clipping_search') }}"><span class="glyphicon glyphicon-search"></span>
-                                      Search</a>
-                                </li>
-                            </ul>
-                        </nav><!--END NAV FOOTER -->
                     </div>
-                    <div class="footer-col-right">
-                        <nav class="nav-footer">
-                            <ul class="menu external-links">
-                                <li class="menu-item dhil">
-                                    <a href="https://www.lib.sfu.ca/help/publish/dh/dhil" target="_blank"><img src="{{ asset('images/dhil-home-page-logo.jpg') }}" /></a>
-                                </li>
-                                <li class="menu-item" sfu>
-                                    <a href="https://www.sfu.ca/" target="_blank"><img src="{{ asset('images/SFU_block_colour_rgb.png') }}" /></a>
-                                </li>
-                            </ul>
-                        </nav><!--END NAV FOOTER -->
-                    </div>
-                </footer>
-            </div><!--END footer-wrapper -->
-        </div><!--END page-wrapper -->
+                {% endfor %}
+            {% endfor %}
+            
+            
+            {% block admin %}
+            
+            {% endblock %}
+            
+            {% block breadcrumb %}
+            
+            {# Check if we're in a nines page #}
+  
+             {% if breadpath is not defined and isNines %}
+                 
+                 {% set breadpath =
+                    { 
+                        0: {
+                            'title': page.title
+                        }
+                    }
+                    
+                  %}
+                  
+              {% endif %}
+              
+              {# Now include the breadcrumb bit #}
+              
+              {% include 'misc/breadcrumb.html.twig' %}
+
+            {% endblock %}
+            
+            {% block pageheader %}
+            
+            {% endblock %}
+            
+            {% block body %}
+                
+            {% endblock %}
+            
+        </main>
+ 
+        <footer class="footer">
+            <div class="container">
+                <div class="footer-menu">
+                    
+                    {# We construct our own menu here
+                    since the knp menu rendering is too wedded
+                    to the default bootstrap assumptions that it's
+                    in the nav bar only #}
+                    
+                    <ul>
```

Comment:
We can define a footer menu with minimal markup if you'd like. Might be good to be consistent with the top nav bar.